test/tools.ts

```diff
@@ -12,6 +18,12 @@ export async function resolveDutOsImage(dutType: string) {
 }
 
 const resolveImageInfo = async (dutType: string) => {
+	if (dutType === 'fincm3') {
+		return {
+			deviceType: 'fincm3',
+			version: await resolveDutOsVersion(),
+		};
+	}
```

Comment:
@Bucknalla Would it be better to implement a switch here for future scalability?